feg/gateway/configs/service_registry.yml

```diff
@@ -57,6 +57,9 @@ services:
   ctraced:
     ip_address: 127.0.0.1
     port: 50079
+  n7_proxy:
+    ip_address: 127.0.0.1
+    port: 9118
```

Comment:
let's place this after csfb. Maybe we find a better position later. Reason is we want to have the ports more or less grouped

Reply:
lets make this 9119 to match `addLocalService(N7_PROXY, 9119)`